docs/source/examples/notebooks/performance/03-pybamm-solvers.ipynb

```diff
@@ -0,0 +1,159 @@
+    "- [`pybamm.ScipySolver`](https://docs.pybamm.org/en/stable/source/api/solvers/scipy_solver.html): This is a wrapper around the `scipy.integrate.solve_ivp` solver. This is generally much slower than the other solvers and is only recommended for very simple problems or for testing.\n",
+    "- [`pybamm.IDAKLUJax`](https://docs.pybamm.org/en/stable/source/api/solvers/idaklu_jax.html) (*experimental*): This is a [JAX](https://jax.readthedocs.io/en/latest/index.html) wrapper around the `pybamm.IDAKLUSolver` solver that allows it to be used within a JAX model. This is still experimental for Windows, so is only recommended for use on Linux or MacOS.\n",
+    "- [`pybamm.JaxSolver`](https://docs.pybamm.org/en/stable/source/api/solvers/jax_solver.html) (*experimental*): This is a pure JAX solver that uses the `jax` library to solve the model equations instead of the Sundials solvers. This is also experimental on Windows, so is only recommended for use on Linux or MacOS.\n",
+    "\n",
+    "The two main solvers are the Casadi and IDAKLU solvers. Out of these two, the IDAKLU solver is generally recommended as it is faster and has more features. Historically, the Casadi solver has been the default solver in PyBaMM, but current development is focused on the IDAKLU solver and this is expected to become the default solver in the near future.\n",
+    "\n",
+    "## Solver features\n",
+    "\n",
+    "- **Speed**: The IDAKLU solver is generally faster than the Casadi solver, both for solving models and also post-processing the ouput variables. However, for some physics options the Casadi solver can be faster, so it is worth trying both.\n",
+    "- **Sensitivity analysis**: The IDAKLU solver can compute forward model sensitivities with respect to input parameters. It uses the Sundials IDAS solver to compute the sensitivities. The Casadi solver can also compute sensitivities, but it is generally slower than the IDAKLU solver and not recommended for this purpose.\n",
+    "- **Adjoint sensitivity analysis**: Either adjoint sensitivity analysis or reverse-mode auto differentiation is useful to compute sensitivities if the number of input parameters is very large (e.g. ML models). Currently only the pure Jax solver supports reverse mode auto differentiation, but adding adjoint sensitivity analysis to the IDAKLU solver is a future development goal.\n",
+    "- **Parallism**: If a list of input parameters are passed to the solver, the IDAKLU solver can solve the model in parallel for each parameter using multiple threads via OpenMP. This can be useful, for example for parameter sweeps. The Casadi solver can also solve models in parallel, but it uses the Python `multiprocessing` library which can be slower than OpenMP. If enough threads are available and the model is large enough, the IDAKLU solver can also parallelise a single solve using OpenMP and the Sundials NVECTOR_OPENMP implementation.\n",
```

Comment:
```suggestion
    "- **Parallelism**: If a list of input parameters are passed to the solver, the IDAKLU solver can solve the model in parallel for each parameter using multiple threads via OpenMP. This can be useful, for example for parameter sweeps. The Casadi solver can also solve models in parallel, but it uses the Python `multiprocessing` library which can be slower than OpenMP. If enough threads are available and the model is large enough, the IDAKLU solver can also parallelise a single solve using OpenMP and the Sundials NVECTOR_OPENMP implementation.\n",
```